packages/ui/src/styles/typography-primitive.css

```
@@ -0,0 +1,42 @@
+@theme {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)